samples/contrib/pytorch-samples/Pipeline-Cifar10-hpo.ipynb

```diff
@@ -216,8 +216,7 @@
    "source": [
     "DEPLOY_NAME=\"torchserve\"\n",
     "MODEL_NAME=\"cifar10\"\n",
-    "ISVC_NAME=DEPLOY_NAME+\".\"+NAMESPACE+\".\"+\"example.com\"\n",
-    "INPUT_REQUEST=\"https://raw.githubusercontent.com/kubeflow/pipelines/master/samples/contrib/pytorch-samples/cifar10/input.json\""
```

Comment:
Don't we need the input for running this notebook?

Reply:
@chauhang The HPO example covers till the training step and those variables were not used.